api/src/db/migrations/v106_renamed_address_city_and_address_municipality.test.ts

```diff
@@ -0,0 +1,27 @@
+import { generateV105ProfileData, generateV105UserData } from "./v105_add_pet_care_essential_question";
+import { migrate_v105_to_v106 } from "./v106_renamed_address_city_and_address_municipality";
+
+describe("migrate_v105_to_v106", () => {
+  it("shows addressCity values as foreignAddressCity and addressMunicipality values as domesticAddressMunicipality", () => {
+    const v105 = generateV105UserData({
+      profileData: generateV105ProfileData({}),
+    });
+    const v106 = migrate_v105_to_v106(v105);
+    const formationFormDataObject = { ...v105.formationData.formationFormData };
+    delete formationFormDataObject.addressCity;
+    delete formationFormDataObject.addressMunicipality;
```

Comment:
Just for some feedback, I think this test could be improved because it's basically just writing the same logic as the tests. I'd prefer a test that is more explicit in its assertions (like, assert that x field and y field are undefined) rather than just using the same code in the test as we do in the real - it's self-reflective